docs/api/paths/diagnostic/keyspace-sync.yaml

```
@@ -7,16 +7,39 @@
 # the file licenses/APL2.txt.
 parameters:
   - $ref: ../../components/parameters.yaml#/keyspace
```

We should include the available query parameters in here (`doc_id`) - appreciate this was already missing before this PR but let's fix it up now.

```suggestion
  - $ref: ../../components/parameters.yaml#/keyspace
  - $ref: ../../components/parameters.yaml#/doc_id
```